### ISSUE TITLE
Add support for receiving calls, DTMF decoding.

### ISSUE THREAD

**Issue by vshymanskyy** [vshymanskyy/TinyGSM#76]
Does anyone have experience with this on SIM800 and A6/A7 ?
Is this functionality needed at all?

**Comment by vshymanskyy** [comment]
PRs are welcome! ;)

**Comment by thbl** [comment]
+1

**Comment by Adney** [comment]
Yes, i need function reseiving calls (AOH), and support dtmf decoding

**Comment by vitzaoral** [comment]
+1

**Comment by atulgo** [comment]
Hi
AT-reference doc for A6 doesnt even list 'AT+DDET' command which is used for DTMF detection.
SIM800L datasheet does list it . You even  get a OK reply for the same . However it does not detect any tone ...
Looking up to you GSM gurus
regards
